packages/domain/lib/src/utils/utility_extensions.dart

```diff
@@ -0,0 +1,14 @@
+extension StringExtension on String {
+  String capitalize() {
+    return "${this[0].toUpperCase()}${this.substring(1).toLowerCase()}";
```

Comment:
what if the string is empty?

Reply:
still actual

Reply:
I added this string:
    if (this.isEmpty) return this;
is this not enough?

Reply:
enough

Reply:
unresolved, because in case of the input "a" u will have crash

Reply:
emmm, no
<img width="862" alt="image" src="https://user-images.githubusercontent.com/111364150/191069461-5ac656dc-faa0-4c83-9547-da3f8f2ef015.png">

Reply:
hm, my bad. resolved